AsyncDisplayKit/Layout/ASLayout.h

```diff
@@ -145,6 +138,14 @@ extern BOOL CGPointIsNull(CGPoint point);
 
 @end
 
+@interface ASLayout (Unavailable)
+
+- (instancetype)init __unavailable;
```

Comment:
There was a big PR that added all of this. We would have to replace it in a bunch of places. To align with other methods let's use `__unavailable` here and we could rethink that in a different PR.